pip/cmdoptions.py

```diff
@@ -498,6 +498,18 @@ def only_binary():
          "options to setup.py install. If you are using an option with a "
          "directory path, be sure to use absolute path.")
 
+download_options = partial(
```

Comment:
Sounds good -- I removed it. Do you think we're almost ready to merge this in? We're blocking on the download features over in #2965 to become available for a feature in `pantsbuild/pants`, and that PR is now blocking on this one.